### DISCUSSION TITLE
Will llama.cpp support NPU in new AI CPUs?

### DISCUSSION THREAD

**Discussion by victorx98** [ggml-org/llama.cpp#8635] [General]
We will have multiple CPUs that are equipped with NPU and more power GPU over 40 TOPS, like Snapdragon X Elite, Intel Lunar lake and AMD Ryzen 9 AI HX 370. Is it possible for llama.cpp to use the combined power from their NPU and GPU?

Thanks

**Comment by hpvd** [comment]
see links to issues in https://github.com/ggerganov/llama.cpp/issues/9181
